demisto_sdk/commands/lint/linter.py

```diff
@@ -1058,7 +1089,7 @@ def _docker_run_pytest(
             int: 0 on successful, errors 1, need to retry 2
             str: Unit test json report
         """
-        log_prompt = f"{self._pack_name} - Pytest - Image {test_image}"
+        log_prompt = f"{self._pack_name} - Pytest - Image {test_image}, docker network disabled = {should_disable_network}"
```

Comment:
for easier reading, please only show `network: enabled/disabled` (instead of True/False for `disabled`)